hack/update-instance-labels.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Replace 'app.kubernetes.io/version: v0.6.x' with 'app.kubernetes.io/version: v0.7.0'
+grep -rl --exclude-dir=hack 'app.kubernetes.io/version: v0.6' ./ | xargs sed -i -E 's/app.kubernetes.io\/version: v0.6(.*)/app.kubernetes.io\/version: v0.7.0/g'
```

Comment:
may want to include the same directories for exclusion as hack/gen-test-targets.sh just so we don't inadvertently change something we don't want to

kfdef|gatekeeper|gcp/deployment_manager_configs|aws/infra_configs|doc|tests|hack|plugins

Reply:
We do want to update tests, right?

Reply:
yep, this does what it needs to then
/lgtm
/approve